metadrive/obs/top_down_obs_multi_channel.py

```diff
@@ -106,16 +115,30 @@ def draw_map(self) -> pygame.Surface:
         self.canvas_background.move_display_window_to(centering_pos)
         self.canvas_road_network.move_display_window_to(centering_pos)
 
-        # self.draw_navigation(self.canvas_navigation)
-        self.draw_navigation(self.canvas_background, (64, 64, 64))
+        if isinstance(self.target_vehicle.navigation, NodeNetworkNavigation):
+            self.draw_navigation_node(self.canvas_background, (64, 64, 64))
+        elif isinstance(self.target_vehicle.navigation, EdgeNetworkNavigation):
+            # TODO: draw edge network navigation
+            pass
+        elif isinstance(self.target_vehicle.navigation, TrajectoryNavigation):
+            self.draw_navigation_trajectory(self.canvas_background, (64, 64, 64))
+
+        if isinstance(self.road_network, NodeRoadNetwork):
+            for _from in self.road_network.graph.keys():
+                decoration = True if _from == Decoration.start else False
+                for _to in self.road_network.graph[_from].keys():
+                    for l in self.road_network.graph[_from][_to]:
+                        two_side = True if l is self.road_network.graph[_from][_to][-1] or decoration else False
+                        LaneGraphics.LANE_LINE_WIDTH = 0.5
+                        LaneGraphics.display(l, self.canvas_background, two_side)
+        elif hasattr(self.engine, "map_manager"):
+            for data in self.engine.map_manager.current_map.blocks[-1].map_data.values():
+                if ScenarioDescription.POLYLINE in data:
+                    LaneGraphics.display_scenario(
```

Comment:
I don't remember what the figure drawn with this code looks like, so I am not sure if `display_scenario` achieves what you want. If not, just check this implementation in the `if semantic_map`: https://github.com/metadriverse/metadrive/blob/cc26984a4c1b77115f6e8b8ed3770d4258895c0d/metadrive/obs/top_down_renderer.py#L52

It gives you filled polygons to show maps and lines to represent lines like this:

![image](https://github.com/metadriverse/metadrive/assets/24489294/69a79362-5b3c-424d-be5e-98f19fd0e94e)